tests/test-sources/plugins/by-name/project-nvim/default.nix

```diff
@@ -1,9 +1,19 @@
 {
   empty = {
-    plugins.project-nvim.enable = true;
+    # Attempts at writing to `datapath`:
+    # ERROR: Invalid `datapath`, reverting to default.
```

Comment:
Should we have another test that configures a _writable_ datapath?

Reply:
I changed all tests to have a writable datapath, except `defaults` for which I disabled `test.runNvim`. It makes more sense I guess.